projects/plugins/boost/app/assets/src/js/pages/getting-started/GettingStarted.svelte

```diff
@@ -0,0 +1,229 @@
+<script>
+	import { derived } from 'svelte/store';
+	import { __ } from '@wordpress/i18n';
+	import ContextInfo from '../../elements/ContextInfo.svelte';
+	import ProductPrice from '../../elements/ProductPrice.svelte';
+	import TemplatedString from '../../elements/TemplatedString.svelte';
+	import Header from '../../sections/Header.svelte';
+	import { getStarted } from '../../stores/config';
+	import config from '../../stores/config';
+	import externalLinkTemplateVar from '../../utils/external-link-template-var';
+	import { getUpgradeURL } from '../../utils/upgrade';
+
+	const pricing = derived( config, $config => $config.pricing );
+
+	const cssOptimizationContext = __(
+		'Move important styling information to the start of the page, which helps pages display your content sooner, so your users don’t have to wait for the entire page to load. Commonly referred to as Critical CSS.',
+		'jetpack-boost'
+	);
+
+	const deferJSContext = __(
+		'Run non-essential JavaScript after the page has loaded so that styles and images can load more quickly. Read more on <link>web.dev</link>.',
+		'jetpack-boost'
+	);
+
+	const lazyLoadingContext = __(
+		'Improve page loading speed by only loading images when they are required. Read more on <link>web.dev</link>.',
+		'jetpack-boost'
+	);
+
+	const supportContext = __(
+		`<p>Paid customers get dedicated email support from our world-class Happiness Engineers to help with any issue.</p><p>All other questions are handled by our team as quickly as we are able to go through the WordPress support forum.</p>`,
+		'jetpack-boost'
+	);
+
+	const webDevLink = 'https://web.dev/';
+
+	// svelte-ignore unused-export-let - Ignored values supplied by svelte-navigator.
+	export let navigate;
+	const chooseFreePlan = () => {
+		getStarted.done();
+		navigate( '/' );
+	};
+
+	const choosePaidPlan = () => {
+		window.location.href = getUpgradeURL();
+	};
+</script>
+
+<div id="jb-settings" class="jb-settings jb-settings--main">
+	<div class="jb-container">
+		<Header />
+	</div>
+
+	<div class="jb-section jb-section--alt jb-section--scores">
+		<div class="jb-container">
+			<div class="comparison-table" style="--rows: 11; --columns: 3">
+				<div class="comparison-table__main">
+					<h1>{__( 'The easiest speed optimization plugin for WordPress', 'jetpack-boost' )}</h1>
+
+					<p class="visible-md comparison-table__feature-row">
+						Optimize CSS Loading
+						<ContextInfo placement="left">{cssOptimizationContext}</ContextInfo>
+					</p>
+					<hr class="visible-md" />
+					<p class="visible-md comparison-table__feature-row">
+						Defer non-essential JavaScript
+						<ContextInfo placement="left">
+							<TemplatedString
+								template={deferJSContext}
+								vars={{
+									...externalLinkTemplateVar( webDevLink ),
+								}}
+							/>
```

Comment:
I don't think it's worth showing a link in a mouse-over popup, as you can't click on it. :)

Reply:
You can :)

Reply:
Well, count me surprised. You can click on the (i) and it stays up so you can click on the links. Nice!

However, is there a reason the links go to the web.dev homepage, and not to the specific articles via the Jetpack redirects (like on the boost dashboard)?